app/views/pages/resources.html.erb

```diff
@@ -110,8 +110,8 @@
           <li><%= link_to 'DD2278 - Application for PPM (DITY) and Counseling Checklist', 'http://www.esd.whs.mil/Portals/54/Documents/DD/forms/dd/dd2278.pdf', class: "usa-external_link" %>: This form will be automatically generated by the Defense Personal Property system upon completion of self counseling for a PPM.</li>
           <li><%= link_to 'SF1038 - Advance of Funds Application (PPM)', 'https://app_gsagov_prod_rdcgwaajp7wr.s3.amazonaws.com/SF1038-77.pdf', class: "usa-external_link" %>: This form needs to be completed and submitted to your local finance office if you will be requesting an advance of funds to help cover some of the expenses associated with your move.</li>
           <li><%= link_to 'DD1351-2 - Travel Voucher', 'http://www.esd.whs.mil/Portals/54/Documents/DD/forms/dd/dd1351-2.pdf', class: "usa-external_link" %>: This form needs to be completed and submitted to your local finance office at the completion of your move so you can claim all your reimbursable expenses that you are entitled to during your move. Some finance offices have online systems for this form in place, however it’s listed here if needed.</li>
-          <li><%= link_to 'Sample Inconvenience Claim Letter', asset_path('claim.pdf'), download: 'claim.pdf' %>: Included is a sample inconvenience claim letter to use as an example. For more information on an Inconvenience claim, please review this <%= link_to 'Frequently Asked Question', faqs_path %>.</li>
-          <li><%= link_to 'PPM Submission Checklist', asset_path('ppm_checklist.pdf'), download: 'ppm_checklist.pdf' %>: The Personally Procured Move checklist will help you gather all the required documents prior to submitting your PPM voucher to the personal property office.</li>
+          <li><%= link_to 'Sample Inconvenience Claim Letter', '/documents/claim.pdf', download: 'Sample Inconvenience Claim Letter' %>: Included is a sample inconvenience claim letter to use as an example. For more information on an Inconvenience claim, please review this <%= link_to 'Frequently Asked Question', faqs_path %>.</li>
+          <li><%= link_to 'PPM Submission Checklist', '/documents/ppm-checklist.pdf', download: 'PPM Submission Checklist' %>: The Personally Procured Move checklist will help you gather all the required documents prior to submitting your PPM voucher to the personal property office.</li>
```

Comment:
We're serving these straight out of `public/documents` and aren't caching them in nginx. That should free us up to update the files, keep the same name, and not sweat outdated cached files.